src/main/cpp/startup_options.cc

```diff
@@ -92,6 +92,7 @@ StartupOptions::StartupOptions(const string &product_name,
       java_logging_formatter(
           "com.google.devtools.build.lib.util.SingleLineFormatter"),
       expand_configs_in_place(true),
+      idle_gc(false),
```

Comment:
Contrary to my earlier statement, I think it's safer to default to the old behavior. Then, we can release it with the toggle and flip the default value in a later release.

Reply:
That makes sense.